### PR TITLE
Dev vcf 40 kb cred syscheck

### PR DESCRIPTION
**Pull request by kannanbalu**

### CODE REVIEW

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 1]
```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
```
Comment (GaryJBlake) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Module Name should be 'Start-VCFUploadBundle'

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
-
-### Syntax
-```
-Start-UploadVCFBundle -json <path to json file>
-```
-
-### Description
-The Start-UploadVCFBundle cmdlet starts upload of bundle(s) to SDDC Manager
```
Comment (GaryJBlake) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Is this from a local file system ?

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 3]
```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
-
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
```
Comment (GaryJBlake) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Module Name should be 'Start-VCFUploadBundle'

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 4]
```diff
@@ -1,38 +0,0 @@
...
-
-### Syntax
-```
-Start-UploadVCFBundle -json <path to json file>
```
Comment (GaryJBlake) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Module Name should be 'Start-VCFUploadBundle'

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 2 above)
Comment (GaryJBlake) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: Module Name should be 'Start-VCFUploadBundle'

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 5]
```diff
@@ -1,38 +0,0 @@
...
-```
-
-### Description
-The Start-UploadVCFBundle cmdlet starts upload of bundle(s) to SDDC Manager
-
-### Examples
-#### Example 1
-```
-Start-UploadVCFBundle -json .\Bundle\bundlespec.json
```
Comment (GaryJBlake) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Module Name should be 'Start-VCFUploadBundle'

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 1 above)
Comment (GaryJBlake) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: Sorry just realized this file has been deleted ignore comments

File: docs/functions/Credentials/Get-VCFCredentialTask.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+```
+Get-VCFCredentialTask -id 7534d35d-98fb-43de-bcf7-2776beb6fcc3
+```
+This example shows how to get the credential tasks for a specific task id
+
+#### Example 3
+```
+Get-VCFCredentialTask -id 7534d35d-98fb-43de-bcf7-2776beb6fcc3 -resourceCredentials
+```
+This example shows how to get resource credentials for a credential task id
```
Comment (GaryJBlake) [inline docs/functions/Credentials/Get-VCFCredentialTask.md]: Description needs to be updated, no mention of the resourceCredentials

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 1 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: yes it's a deleted file

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 2 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: same as above. It's a deleted file

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 3 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: same as above. It's a deleted file

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 4 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: It's a deleted file

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 5 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: It's a deleted file

**Pull request merged**